src/handlers/event.handler.ts

```
@@ -55,11 +55,11 @@ export class EventHandler {
       return v;
     };
 
-    // Single table flow: directly save into Events for each repetition
+    // Single table flow: upsert into Events keyed on eventRepetitionId
     await Promise.all(
       data.eventRepetitionData.map(async (eventRepetitionData) => {
         await this.dbService.saveEventData({
-          eventDetailId: coerceString(data.eventDetailsData.eventDetailId)!,
+          eventRepetitionId: coerceString(eventRepetitionData.eventRepetitionId)!,
```

ESLint

[error] Replace `eventRepetitionData.eventRepetitionId` with `⏎············eventRepetitionData.eventRepetitionId,⏎··········`

(prettier/prettier)